packages/shared/web/src/dashboard-view/hooks/use-liquidity-leveraged-grid.tsx

```diff
@@ -28,6 +31,47 @@ export const useLiquidityLeveragedGrid = (
         (n) => n.asset.balance.underlying.symbol === y.underlying.symbol
       );
 
+      const handleBottomValue = () => {
```

Comment:
You shouldn't use inline functions like this, this creates a performance overhead. You should just put the logic inline without creating a function or make a function outside of the hook.

Reply:
You do not need to use `marketYield` in here. You can use `y` regardless of currentPosition.